zuul.d/projects.yaml

```diff
@@ -56,7 +56,6 @@
       - cifmw-molecule-set_openstack_containers
       - cifmw-molecule-tempest
       - cifmw-molecule-test_deps
-    name: openstack-k8s-operators/ci-framework
-    templates:
-    - podified-multinode-edpm-pipeline
-    - data-plane-adoption-pipeline
+#    name: openstack-k8s-operators/ci-framework
```

Comment:
I guess this is commented out for faster testing, just leaving a comment so that it does not slip by :)

Reply:
right thanks for checking